app/analytics/routes.py

```diff
@@ -107,8 +107,11 @@ def datasets_views():
                     v.nb_uniq_visitors if v.nb_uniq_visitors is not None else 0)
 
         if not exists and v.dataset_id is not None:
+            dataset_info = Dataset.query.filter_by(dataset_id=v.dataset_id).first()
+            print(dataset_info)
```

Comment:
We probably don't want this print statement

Reply:
oooooops